### ISSUE TITLE
Is it working as of Nov 2013?

### ISSUE THREAD

**Issue by regstuff** [adamzr/RSS-Alert#6]
I installed it and loaded up a bunch of busy feeds but I don't get any notifications. Is the extension still working? I'm using chrome v31 btw.

**Comment by adamzr** [comment]
No, the problem is that HTML web notifications were disabled. The notifications need to be changed to plain text.

**Comment by regstuff** [comment]
Thanks for the info.

**Comment by adamzr** [comment]
I've pushed out an update to fix this. The notifications will look different, but should be back now.
